src/Tizen.NUI/src/public/Theme/Theme.cs

```diff
@@ -27,10 +28,29 @@ namespace Tizen.NUI
 {
     /// <summary></summary>
     [EditorBrowsable(EditorBrowsableState.Never)]
-    public class Theme : BindableObject
+    public class Theme : BindableObject, IResourcesProvider
     {
         private readonly Dictionary<string, ViewStyle> map;
         private string baseTheme;
+        private string resource;
+
+        private string XamlFile { get; set; }
```

Comment:
Could it be a `Field`?  Since it is `private` and has default getter and setter.